client/src/js/app.js

```diff
@@ -102,6 +102,12 @@ function bhimaconfig($routeProvider) {
     templateUrl : 'partials/reports_proposed/invoice/invoice.html'
   })
 
+  /* Admin : depot management */
+  .when('/depots_management', {
```

Comment:
There is already a '/depots' route which is used for the 'stock' management module